www/source/tutorials/build-your-own/connect-database.html.slim

```diff
@@ -0,0 +1,284 @@
+---
+title: Habitat - Connect to Database
+---
+
+section
```

Comment:
This page would be hugely improved with a table of contents linking to all of the sub-articles below.

Reply:
I included one for this topic and two others that felt a bit long.